docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -39,118 +47,146 @@ The following tags should be added to the virtual private network (VPC) public s
 
 ## Deploy an AWS Cluster
 
-Use the following steps to provision a new AWS EKS cluster:
+Use the following steps to deploy an AWS cluster in which to provision an EKS cluster.

```

Comment:
I don't think this is right. The following steps will deploy an EKS cluster on AWS (or similar).

Reply:
Fixed.